framework/platform/bpl/include/bpl_cfg.h

```diff
@@ -260,6 +265,15 @@ struct BPL_WLAN_PARAMS {
     char security[BPL_SEC_LEN];
 };
 
+/* DCS Parameters */
```

Comment:
I think we should add comments explaining each parameter, like done in `BPL_WLAN_PARAMS` for example

Reply:
I agree, but we can do it post-merge.
Created a task #460 for updating the documentation.

Reply:
done